ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -67,7 +67,7 @@ const PersonalSignInfo: React.FC = () => {
           alertKey="requestFrom"
           ownerId={currentConfirmation.id}
           label={t('requestFrom')}
-          tooltip={t('requestFromInfo')}
```

Comment:
note: we will want to include these nuances in our upcoming tests https://github.com/MetaMask/metamask-extension/issues/24679